lldb/source/Symbol/Symbol.cpp

```diff
@@ -262,14 +263,12 @@ void Symbol::GetDescription(Stream *s, lldb::DescriptionLevel level,
   }
   if (ConstString demangled = m_mangled.GetDemangledName()) {
     s->PutCString(", name=\"");
-    s->PutCStringColorHighlighted(demangled.GetStringRef(), pattern,
-                                  ansi_prefix, ansi_suffix);
+    s->PutCStringColorHighlighted(demangled.GetStringRef(), pattern_info);
     s->PutCString("\"");
   }
   if (ConstString mangled_name = m_mangled.GetMangledName()) {
     s->PutCString(", mangled=\"");
-    s->PutCStringColorHighlighted(mangled_name.GetStringRef(), pattern,
-                                  ansi_prefix, ansi_suffix);
+    s->PutCStringColorHighlighted(mangled_name.GetStringRef(), pattern_info);
```

Comment:
The name `pattern_info` for a local variable is fine I think, the name of the type bothers me more.

Reply:
Since the struct name changed, should we rename this to something like `highlight_settings` or `color_settings`?

Reply:
`highlight_settings`, given that you could use any ANSII sequence. I'm sure you could turn the text upside down if you really wanted to :)